templates/azure_creds_file.j2

```diff
@@ -0,0 +1,5 @@
+[default]
+subscription_id={{ azure_subscription_id }}
+client_id={{ azure_sp_client_id }}
+secret={{ azure_sp_password }}
+tenant={{ azure_tenant_id }}
```

Comment:
At the point that this template is used, I only see that `azure_sp_password` and `azure_sp_client_id` are defined. Where do `azure_tenant_id` and `azure_subscription_id` come from? If those are role input vars, they should be properly namespaced with the role name, and included in the README.